velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -35,15 +35,27 @@ namespace facebook::velox::connector::hive::iceberg {
 class HiveIcebergTest : public HiveConnectorTestBase {
  public:
   void assertPositionalDeletes(
-      const std::vector<int64_t>& deleteRows,
+      const std::vector<std::vector<int64_t>>& deleteRowVectors,
       bool multipleBaseFiles = false) {
+    std::vector<int64_t> deleteRows;
+    uint32_t numDeleteRows = 0;
+    for (const std::vector<int64_t>& deleteRowVector : deleteRowVectors) {
+      numDeleteRows += deleteRowVector.size();
+    }
+
+    deleteRows.reserve(numDeleteRows);
```

Comment:
Would it still work if we remove line 41-46? This is just a test, no need to calculate numDeleteRows and reserve the space for performance reasons.

Reply:
Yes, it will work, I will remove this.